tests/e2e/LicenseCheckerCase.php

```diff
@@ -158,23 +138,17 @@ public function test_report_not_allowed_licenses(): void
             ->withLicense('MIT')
             ->build();
 
-        $exitCode = $this->commandTester->execute(['--allow-file' => $allowFile, '--path' => $this->path]);
-        $output = $this->getOutputLines();
-
-        self::assertSame(1, $exitCode);
-        self::assertCount(8, $output);
-        self::assertSame(
-            '[ERROR] Dependency "lendable/apache" has license "Apache-2.0" which is not in the allowed list.',
-            $output[3],
-        );
-        self::assertSame(
-            '[ERROR] Dependency "lendable/bsd3_mit" has license "BSD-3-Clause" which is not in the allowed list.',
-            $output[5],
-        );
-        self::assertSame(
-            '[ERROR] Dependency "package/bsd3" has license "BSD-3-Clause" which is not in the allowed list.',
-            $output[7],
-        );
+        $this->commandTester->execute(['--allow-file' => $allowFile, '--path' => $this->path]);
+        $this->getOutputLines();
```

Comment:
I elieve this method can be dropped now?

Reply:
Yeah I missed this, done in https://github.com/Lendable/composer-license-checker/pull/83/commits/7d33e5ef35c2dd51473a0211fba08fafe8c53ea6